app/controllers/teacher_sets_controller.rb

```diff
@@ -11,47 +11,44 @@ class TeacherSetsController < ApplicationController
   # Called on loading the teacher set list page and when the user selects
   # a facet to filter by.
   def index
-    LogWrapper.log('DEBUG', {'message' => 'index.start', 'method' => 'app/controllers/teacher_sets_controller.rb.index'})
-    begin
-      # Feature flag: 'teacherset.data.from.elasticsearch.enabled = true' means gets teacher-set documents from elastic search.
-      # teacherset.data.from.elasticsearch.enabled = false means gets teacher-set data from database.
-      if MlnConfigurationController.new.feature_flag_config('teacherset.data.from.elasticsearch.enabled')
-        LogWrapper.log('INFO', {'message' => "Calling elastic search to get teacher-sets", 
-                                'method' => 'app/controllers/teacher_sets_controller.rb.index'})
-    
-        # Get teachersets and facets from elastic search
-        teacher_sets, @facets = ElasticSearch.new.get_teacher_sets_from_es(params)
-        @teacher_sets = teacher_sets_from_elastic_search_doc(teacher_sets)
-      else
-        LogWrapper.log('INFO', {'message' => "Calling database to get teacher-sets", 
-                                'method' => 'app/controllers/teacher_sets_controller.rb.index'})
-        @teacher_sets = TeacherSet.for_query params
-        @facets = TeacherSet.facets_for_query @teacher_sets
-      end
-      # Determine what facets are selected based on query string
-      @facets.each do |f|
-        f[:items].each do |v|
-          k = f[:label].underscore
-          v[:selected] = params.keys.include?(k) && params[k].include?(v[:value].to_s)
-        end
+    # Feature flag: 'teacherset.data.from.elasticsearch.enabled = true' means gets teacher-set documents from elastic search.
+    # teacherset.data.from.elasticsearch.enabled = false means gets teacher-set data from database.
+    if MlnConfigurationController.new.feature_flag_config('teacherset.data.from.elasticsearch.enabled')
+      LogWrapper.log('INFO', {'message' => "Calling elastic search to get teacher-sets", 
```

Comment:
This is a nice refactor! Thanks

Do we need log statements and the one on line 24? I'm fine if they're useful but if not let's drop them.